src/entities/index.ts

```diff
@@ -1,8 +1,9 @@
-export { SignedOrderEntity } from './SignedOrderEntity';
-export { TransactionEntity } from './TransactionEntity';
 export { KeyValueEntity } from './KeyValueEntity';
-export { TransactionEntityOpts } from './types';
-export { PersistentSignedOrderEntity } from './PersistentSignedOrderEntity';
 export { MakerBalanceChainCacheEntity } from './MakerBalanceChainCacheEntity';
-export { SignedOrderV4Entity } from './SignedOrderV4Entity';
+export { PersistentSignedOrderEntity } from './PersistentSignedOrderEntity';
 export { PersistentSignedOrderV4Entity } from './PersistentSignedOrderV4Entity';
+export { RfqmQuoteEntity } from './RfqmQuoteEntity';
+export { SignedOrderEntity } from './SignedOrderEntity';
```

Comment:
just a re-ordering?

Reply:
yup, just sorted alphabetically and added the `RfqmQuoteEntity`